### PR TITLE
Bug 1289333 - Set crash signature and related fields

### PR DESCRIPTION
**Pull request by KWierso**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2294)

### CODE REVIEW

**Comment by KWierso** [comment]
This should work. Before this lands, I still need to add it to the tests and figure out how to show the new crash signature field more nicely.

File: tests/webapp/api/test_bugzilla.py [hunk 1]
```diff
@@ -45,7 +45,61 @@ def request_callback(request):
...
             "comment": "Intermittent Description",
             "comment_tags": "treeherder",
-            "keywords": "intermittent-failure",
+            "keywords": ["intermittent-failure"],
+        }
+    )
+
+    content = json.loads(resp.content)
+
+    print content
```
Comment (camd) [inline tests/webapp/api/test_bugzilla.py]: I presume you want to remove the prints here

File: tests/webapp/api/test_bugzilla.py [hunk 2]
```diff
@@ -45,7 +45,61 @@ def request_callback(request):
...
+def test_create_crash_bug(webapp, eleven_jobs_stored, activate_responses, test_user):
+    """
+    test successfully creating a bug with a crash signature in bugzilla
+    """
+
+    def request_callback(request):
+        headers = {}
+        requestdata = json.loads(request.body)
+        requestheaders = request.headers
+        print requestdata
```
Comment (camd) [inline tests/webapp/api/test_bugzilla.py]: leftover prints?

File: ui/js/controllers/bugfiler.js [hunk 3]
```diff
@@ -254,11 +255,20 @@ treeherder.controller('BugFilerCtrl', [
                 descriptionStrings += $scope.modalComment;
             }
 
-            var keywords = $scope.isIntermittent ? "intermittent-failure" : "";
+            var keywords = [];
```
Comment (camd) [inline ui/js/controllers/bugfiler.js]: I actually preferred your previous way.  :):

    var keywords = $scope.isIntermittent ? ["intermittent-failure"] : [];

File: ui/partials/main/intermittent.html [hunk 4]
```diff
@@ -76,6 +76,11 @@
         <input type="text" ng-model="modalDependsOn" placeholder="Depends on" uib-tooltip="Comma-separated list of bugs" tooltip-placement="bottom" />
         <input type="text" ng-model="modalSeeAlso" placeholder="See also" uib-tooltip="Comma-separated list of bugs" tooltip-placement="bottom" />
       </div>
+
+      <div ng-hide="crashSignatures.length === 0" id="modalCrashSignatureDiv">
```
Comment (camd) [inline ui/partials/main/intermittent.html]: I think you could just do:

    <div ng-hide="!crashSignatures.length"

or:

    <div ng-show="crashSignatures.length"

**Pull request merged**